lib/__tests/parse.js

```diff
@@ -371,6 +372,71 @@ describe('parse', () => {
         });
     });
 
+    describe('onToken', () => {
+        const source = '/*123*/.foo[a=/* 234 */] {\n  color: red;} /*567*';
+
+        it('as function', () => {
+            const actual = [];
+            parse(source, {
+                onToken(type, start, end) {
+                    actual.push({ type, start, end });
+                }
+            });
+
+            assert.deepStrictEqual(actual, [
```

Comment:
Nit: The array can be stored as `expected` right after `const source = '...'` to reduce repetition, since it's the same for both test cases.